package.json

```diff
@@ -13,7 +13,7 @@
   "license": "GPL3",
   "dependencies": {
     "@rsksmart/bridge-state-data-parser": "^1.2.0",
-    "@rsksmart/bridge-transaction-parser": "^1.1.4",
+   "@rsksmart/bridge-transaction-parser": "github:rsksmart/bridge-transaction-parser#fbdeb686fc438f530d3a5f01bce062fd455be01f",
```

Comment:
Why this? Do we need a new release for `bridge-transaction-parser`?

Reply:
Yes. We need to update the `@rsksmart/rsk-precompiled-abis` dependency in the  `bridge-transaction-parser` library. Check: https://github.com/rsksmart/bridge-transaction-parser/compare/master...use-lovel-precompiled-abis-commit